central/role/datastore/defaults.go

```diff
@@ -0,0 +1,220 @@
+		description: "For automation: it consists of the permissions to create Sensors in secured clusters",
+		resourceWithAccess: []permissions.ResourceWithAccess{
+			permissions.View(resources.Cluster),
+			permissions.Modify(resources.Cluster),
+			permissions.Modify(resources.Administration),
+		},
+	},
+	accesscontrol.VulnMgmtApprover: {
+		idSuffix:    "vulnmgmtapprover",
+		postgresID:  accesscontrol.DefaultPermissionSetIDs[accesscontrol.VulnMgmtApprover],
+		description: "For users: use it to provide access to approve vulnerability deferrals or false positive requests",
+		resourceWithAccess: []permissions.ResourceWithAccess{
+			permissions.View(resources.VulnerabilityManagementApprovals),
+			permissions.Modify(resources.VulnerabilityManagementApprovals),
+		},
+	},
+	accesscontrol.VulnMgmtRequester: {
+		idSuffix:    "vulnmgmtrequester",
+		postgresID:  accesscontrol.DefaultPermissionSetIDs[accesscontrol.VulnMgmtRequester],
+		description: "For users: use it to provide access to request vulnerability deferrals or false positives",
+		resourceWithAccess: []permissions.ResourceWithAccess{
+			permissions.View(resources.VulnerabilityManagementRequests),
+			permissions.Modify(resources.VulnerabilityManagementRequests),
+		},
+	},
+	// TODO ROX-13888 when we migrate to WorkflowAdministration we can remove VulnerabilityReports and Role resources
+	accesscontrol.VulnReporter: {
+		idSuffix:    "vulnreporter",
+		postgresID:  accesscontrol.DefaultPermissionSetIDs[accesscontrol.VulnReporter],
+		description: "For users: use it to create and manage vulnerability reporting configurations for scheduled vulnerability reports",
+		resourceWithAccess: func() []permissions.ResourceWithAccess {
+			if !env.PostgresDatastoreEnabled.BooleanSetting() {
+				return []permissions.ResourceWithAccess{
+					permissions.View(resources.Access),                 // required for scopes
+					permissions.View(resources.Integration),            // required for vuln report configurations
+					permissions.View(resources.VulnerabilityReports),   // required for vuln report configurations prior to collections
+					permissions.Modify(resources.VulnerabilityReports), // required for vuln report configurations prior to collections
+				}
+			}
+			return []permissions.ResourceWithAccess{
+				permissions.View(resources.WorkflowAdministration),   // required for vuln report configurations
+				permissions.Modify(resources.WorkflowAdministration), // required for vuln report configurations
+				permissions.View(resources.Integration),              // required for vuln report configurations
+			}
+		}(),
+	},
+	accesscontrol.VulnerabilityManagementConsumer: {
+		idSuffix:    "vulnmgmtconsumer",
+		postgresID:  accesscontrol.DefaultPermissionSetIDs[accesscontrol.VulnerabilityManagementConsumer],
+		description: "For users: use it to provide read-only access to analyze vulnerabilities and initiate risk acceptance process",
```

Comment:
Shall we also mention the report configuration, i.e. `view report configurations`?

Reply:
The language is kept `analyze and assess` on purpose. This allows us to keep it generic and avoid revisiting it in the future when this default permission set includes access to another sub-feature.

P.S. Watch images is another current sub-feature.